test/test_product_config.py

```diff
@@ -491,14 +504,42 @@ def test_from_config_narrowband(
         )
         assert acv.sources(0) == tuple(src_streams[0:2])
         assert acv.sources(1) == tuple(src_streams[2:4])
-        assert acv.data_rate(1.0, 0) == 27392e6 * 2 / 8
+        assert acv.data_rate(1.0, 0) == 27392e6 * 2 / DECIMATION
         assert acv.input_labels == narrowband_config["src_streams"]
         assert acv.w_cutoff == 1.0  # Default value
-        assert acv.window_function is None
+        assert acv.window_function == defaults.WINDOW_FUNCTION
         assert acv.pfb_taps == defaults.PFB_TAPS
         assert acv.dither is None
         assert acv.command_line_extra == []
 
+    def test_from_config_narrowband_vlbi(
+        self,
+        narrowband_vlbi_config: Dict[str, Any],
+        src_streams: List[DigBasebandVoltageStreamBase],
+    ) -> None:
+        acv = GpucbfAntennaChannelisedVoltageStream.from_config(
+            Options(), "vlbi1_acv", narrowband_vlbi_config, src_streams, {}
+        )
+        # Most attributes we don't bother testing - just those affected by VLBI mode
+        assert acv.bandwidth == src_streams[0].adc_sample_rate / 2 / DECIMATION
+        assert acv.n_samples_between_spectra == 2 * narrowband_vlbi_config["n_chans"] * DECIMATION
+        assert acv.data_rate(1.0, 0) == 27392e6 * 2 / DECIMATION
+        assert acv.w_cutoff == 0.0  # Default value for VLBI
+        assert acv.window_function == defaults.WINDOW_FUNCTION_VLBI
+        assert acv.pfb_taps == 1
+
+    def test_vlbi_bad_pass_bandwidth(
+        self,
+        narrowband_vlbi_config: Dict[str, Any],
+        src_streams: List[DigBasebandVoltageStreamBase],
+    ) -> None:
+        bandwidth = src_streams[0].adc_sample_rate / 2 / DECIMATION
+        narrowband_vlbi_config["narrowband"]["vlbi"]["pass_bandwidth"] = bandwidth
+        with pytest.raises(ValueError):
+            GpucbfAntennaChannelisedVoltageStream.from_config(
+                Options(), "vlbi1_acv", narrowband_vlbi_config, src_streams, {}
+            )
```

Comment:
Argh, I'm going to have to be a small pain and request an update to this `pytest.raises` statement.

I was wondering why it was looking so meek. 
* There are some tests where the name doesn't exactly tell me/us what failure case it's checking
  * e.g. `test_n_chans_not_power_of_two` is explicit enough, but 
  * `test_narrowband_bad_centre_frequency` includes the `match={err_string}` to report why ~exactly the arg was bad
* I would request the same treatment here to say why `bandwidth` is bad for `pass_bandwidth`
  * If I need to motivate my request more, I had to go back to `product_config.py` to check why `bandwidth` is bad.
  * I feel like the test should have most/all of the information I need.